src/commands/cmd_stream.cc

```diff
@@ -472,6 +488,82 @@ class CommandXInfo : public Commander {
 
     return Status::OK();
   }
+
+  Status getGroupInfo(Server *srv, Connection *conn, std::string *output) {
+    redis::Stream stream_db(srv->storage, conn->GetNamespace());
+    std::vector<std::pair<std::string, StreamConsumerGroupMetadata>> result_vector;
+    auto s = stream_db.GetGroupInfo(args_[2], result_vector);
+    if (!s.ok() && !s.IsNotFound()) {
+      return {Status::RedisExecErr, s.ToString()};
+    }
+
+    if (s.IsNotFound()) {
+      return {Status::RedisExecErr, errNoSuchKey};
+    }
+
+    if (result_vector.empty()) {
+      output->append(redis::SimpleString("(empty array)"));
```

Comment:
For empty array should return the RESP empty array instead literal "(empty array)".

Refer https://github.com/redis/redis/blob/15a048d4f0465dc58be21d7e46d527ccfcc34a64/src/t_stream.c#L3907